webapp/templates/jury/problems.html.twig

```diff
@@ -40,49 +38,13 @@
 
 {% block extrafooter %}
     {{ parent() }}
+    <script src="{{ asset('js/multi-delete.js') }}"></script>
```

Comment:
Why can't we add this in the head?

Reply:
Could be in `extrahead` I assume (didn't test yet), but having it here makes it close to the `initializeMultiDelete` call, which needs to happen after the DOM has been inserted.

Is there a benefit of moving it from `extrafooter` to `extrahead`?

Reply:
I prefer to see all loaded scripts in the head, so consistency.

But feel free to leave it there. I'll check in the future if I care enough about it to move it myself.